.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Go
+on: [push]
+jobs:
+  run:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    env:
+      OS: ${{ matrix.os }}
+    steps:
+      - name: Set up Go
```

Comment:
also, very minor point, we should be able to optionally remove the `name` and `id` sections on some of the more 'boring' tasks, and just let it use defaults; the `name` and `id` of course is just stylistic, just putting it out there

Reply:
See comments around name already.

Reply:
yeah, apologies, I didn't need to make the same comment on all locations 🥴